x-pack/legacy/plugins/index_management/public/app/sections/home/index_list/index_actions_context_menu/index_actions_context_menu.js

```diff
@@ -211,7 +216,20 @@ export class IndexActionsContextMenu extends Component {
       },
     });
     getActionExtensions().forEach(actionExtension => {
-      const actionExtensionDefinition = actionExtension(indices, reloadIndices);
+      const actionExtensionDefinition = actionExtension({
```

Comment:
@sebelga Just a head's up that a few files are changing in Index Management, which we'll need to reconcile.

If I merge this PR in before you finish migrating Index Management then you'll just need to reconcile the conflicts by accepting these changes but passing in the NP http service in lieu of `getNewPlatformCompatibleHttpClient`.

If you merge in the Index Management migration before I merge this PR then I'll be able to reconcile the conflicts myself, but I don't think that's the best way to proceed. It would create less work for you to wait for this PR to get merged first, because then you won't need to preserve the dependency upon Angular's `$http` service.

Reply:
OK no problem, thanks for the heads up @cjcenizal !